spec/requests/profiles_spec.rb

```diff
@@ -129,6 +129,18 @@
           expect(response).to redirect_to(profile_url(profile))
         end
       end
+
+      context "with admin" do
+        let(:user) { create :user, :admin }
+
+        include_examples "unauthorized access"
+
+        it "does not update profile" do
```

Comment:
Small nitpick (as mine all are)
```suggestion
        it "does not update the profile" do
```